docs/backup_and_restore.rst

```diff
@@ -160,7 +160,7 @@ backup:
 .. code:: sh
 
    cd install_files/ansible-base
-   ansible-playbook -i inventory -u <SSH username> -K -t backup securedrop-prod.yml -e restore_file="<your backup archive filename>"
+   ansible-playbook -u <SSH username> -K -t backup securedrop-prod.yml -e restore_file="<your backup archive filename>"
```

Comment:
This isnt particular to this line, but I think you should add logic to also abstract away the ssh_username and just read it from the site-specific vars. I would be fine if that was moved to another PR also. Thoughts?